cpmonitor/models.py

```diff
@@ -144,7 +142,6 @@ class Meta:
         related_name="cap_checklist",
     )
 
-    cap_exists = models.BooleanField("Gibt es einen KAP?", default=False)
```

Comment:
I think we really need this. You are right, that most other entries refer to a KAP and some do not make any sense without one. However there are cities without a KAP and some of their local teams want to do the monitoring. (Bargdeheide / Tom) Some of the other checklist items might still be true if there is no KAP and some might be rephrased to cover cases without a KAP, as well. Could you please leave it in, for now?